types/hooks/screens/useAnonymousChatInfoScreenHook.types.ts

```diff
@@ -1,9 +1,23 @@
 import {ChannelList, ChannelListMemberSchema} from '../../database/schema/ChannelList.types';
+import {Member} from '../../database/schema/ChatListDetail.types';
 
 interface UseAnonymousChatInfoScreenHook {
   channelInfo: ChannelList;
   goBack: () => void;
-  onContactPressed: (item: ChannelListMemberSchema) => void;
+  onContactPressed: (item: ChannelListMemberSchema, from?: string) => void;
+  isAnonymous: boolean;
+  selectedUser: ChannelListMemberSchema;
+  showPopupBlock: boolean;
+  handleClosePopup: () => void;
+  isAnonymousUser: (item: any) => void;
+  handlePressPopup: (status: string) => void;
+  handleCloseSelectUser: () => void;
+  openModal: boolean;
+  isAnonymousModalOpen: boolean;
+  blockModalRef: () => void;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `blockModalRef` function is declared as returning void and doesn't accept any parameters. If this function is supposed to be a reference to a modal, it should probably be of type `React.RefObject<ModalType>` where `ModalType` is the type of your modal component.
```diff
-   blockModalRef: () => void;
+   blockModalRef: React.RefObject<ModalType>;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->